views/pages/index.ejs

```diff
@@ -3,16 +3,37 @@
 
 <head>
     <%- include('../layout/head') %>
+   
 </head>
 
 <body>
     <header>
-        <%- include('../layout/header') %>
+       <%- include('../layout/header') %>
+       <%- include('partials/searchBar') %>
     </header>
-    <main>
-        <%- include('partials/searchBar') %>
-        <%- include('partials/favoriteArtists') %>
-        <%- include('partials/cities') %>
+    <main class='indexMain'>
+       <div class='favoritesPartial'>
+        <h2 class='favorites'>Favorite Artists</h2>
+        <ul class= 'favArtists'>
+            <% artists.forEach(nameOfArtist => { %>
```

Comment:
Hey this undoes some of the work we’ve done. Can you move the changes in classes to the artwork partial since that’s what we are using now?